composer.json

```diff
@@ -11,24 +10,9 @@
         "apiary"
     ],
     "homepage": "https://api-tools.getlaminas.org",
-    "support": {
-        "docs": "https://api-tools.getlaminas.org/documentation",
-        "issues": "https://github.com/laminas-api-tools/api-tools-documentation-apiblueprint/issues",
-        "source": "https://github.com/laminas-api-tools/api-tools-documentation-apiblueprint",
-        "rss": "https://github.com/laminas-api-tools/api-tools-documentation-apiblueprint/releases.atom",
-        "chat": "https://laminas.dev/chat",
-        "forum": "https://discourse.laminas.dev"
-    },
-    "config": {
-        "sort-packages": true
-    },
-    "extra": {
-        "laminas": {
-            "module": "Laminas\\ApiTools\\Documentation\\ApiBlueprint"
-        }
-    },
+    "license": "BSD-3-Clause",
```

Comment:
We do not want another normalization because it was already done during the migration from ZF.
Please revert all related changes. Thanks!

Reply:
Sorry I'm unsure what you're referring to? This PR should just add the upgrade to latest laminas cs, upgrade phpunit to 9.5, and allow PHP. If you mean running composer-normalize, this has not been done here?